src/box/memtx_rtree.cc

```diff
@@ -146,20 +146,139 @@ extract_rectangle(struct rtree_rect *rect, struct tuple *tuple,
 /* {{{ MemtxRTree Iterators ****************************************/
 
 struct index_rtree_iterator {
-        struct iterator base;
-        struct rtree_iterator impl;
+	struct iterator base;
+	struct rtree_iterator impl;
 	/** Memory pool the iterator was allocated from. */
 	struct mempool *pool;
+	/* last fetched tuple (pos). Is used for pagination */
```

Comment:
Please, start with a capital letter and finish with a dot : `/* A comment. */`
Also, what does `pos` mean here?
Also, please add the second `*` symbol at the beginning of the comment.